ts/src/base/Exchange.ts

```diff
@@ -2507,7 +2509,7 @@ export default class Exchange {
         } catch (e) {
             errorMessage = e.toString ();
         }
-        throw new NotSupported (this.id + ' ' + method + '() failed to fetch correct data from website. Probably webpage markup has been changed, breaking the page custom parser.' + errorMessage);
+        throw new NotSupported (this.id + ' ' + method + '() failed to fetch correct data from website. Probably webpage markup has been changed, breaking the page custom parser. ' + errorMessage + ' | ' + rawContent);
```

Comment:
I don't think we should print the rawContent here, it might be a huge piece of unreadable HTML, json , etc

Reply:
but it would happen only once (when it would break build) and we would have to anyway fix it, and might be useful to spot the issue on first case? no?

Reply:
@ttodua I don't think so, because it will be unformatted, so most likely we won't be able to spot it right away. Besides that the user might be getting this error as well due to network conditions/location/etc

Reply:
this is also fixed

Reply:
bigone passes tests too